src/apis/emscripten/time.rs

```diff
@@ -179,8 +180,6 @@ pub extern "C" fn _localtime(time_p: u32, instance: &mut Instance) -> c_int {
         (*tm_struct_ptr).tm_wday = result_tm.tm_wday;
         (*tm_struct_ptr).tm_yday = result_tm.tm_yday;
         (*tm_struct_ptr).tm_isdst = result_tm.tm_isdst;
-        (*tm_struct_ptr).tm_gmtoff = result_tm.tm_gmtoff as i32;
-        (*tm_struct_ptr).tm_zone = copy_cstr_into_wasm(instance, result_tm.tm_zone) as i32;
```

Comment:
Any reason on why we are skipping those?

Reply:
yeah these fields are gnu library extensions. The `time` crate doesn't read them. We could still leave the fields, but leave them with default values. Do you have a preference? We could also conditionally compile using `time` crate for windows only. Thoughts?

Reply:
I think it might be good to preserve the fields for reference in the future, in case we move out of the `time` crate, or in case something fails in the future regarding gmt_off or timezone.

I'm not that concerned about using `time` only for windows, as it might make the code a bit uglier with not that much gain.